test/scalacheck/scala/reflect/quasiquotes/TermConstructionProps.scala

```diff
@@ -232,13 +232,14 @@ object TermConstructionProps extends QuasiquoteProperties("term construction") {
     object O
     implicit val liftO = Liftable[O.type] { _ => q"foo; bar" }
     assertEqAst(q"f(..$O)", "f(foo, bar)")
-  }
+  }: @annotation.nowarn("cat=unused-locals")
```

Comment:
@som-snytt seems like a false positive, wdyt?

Reply:
that looks familiar but I can't find a ticket.